.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+
+references:
+  workspace_root: &workspace_root ~/project
+
+  attach_workspace: &attach_workspace
+    attach_workspace:
+      at: *workspace_root
+
+  filters_tags: &filters_tags
+    filters:
+      tags:
+        only: /.*/
+
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          cache-key: yarn.lock
+          dir: ~/.cache/yarn
+          steps:
+            - run: yarn --frozen-lockfile --ignore-scripts
+            - run: yarn lerna bootstrap
+            - run: yarn lerna run build --concurrency=2
+      - persist_to_workspace:
+          root: *workspace_root
+          paths: .
+
+  test:
+    executor: node/default
+    steps:
+      - *attach_workspace
+      - run: yarn lint
+      - run: yarn format
+      - run: yarn tsc
+      - run: yarn test:all --coverage --runInBand
+      - run: '[ $COVERALLS_REPO_TOKEN ] && yarn coveralls < demo/coverage/lcov.info || true'
+
+  deploy:
+    executor: node/default
+    environment:
+      NODE_DEBUG: gh-pages
+    steps:
+      - *attach_workspace
+      - run: yarn lerna run build:demo --concurrency=2
+      - run: utils/scripts/gh-pages.js
+
+  publish:
+    executor: node/default
+    steps:
+      - *attach_workspace
+      - run: npm set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+      - run: yarn lerna publish from-git --ignore-scripts --no-verify-access --yes
+
+workflows:
+  main:
+    jobs:
+      - build:
+          <<: *filters_tags
```

Comment:
Why do we have to filter tags for the `build/test` jobs only?

Reply:
> if a job requires any other jobs (directly or indirectly), you must specify tag filters for those jobs

https://circleci.com/docs/2.0/configuration-reference/#tags

Reply:
Oops, this is carry-over from the linting repos. We don't need tag filtering in the lerna workflows.